packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -77,6 +77,12 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
         return this._usedRoutes;
     }
 
+    // If this node is marked as unreferenced, the time when it marked as such.
+    private _unreferencedTimestamp: number | undefined;
+    public get unreferencedTimestamp(): number | undefined {
```

Comment:
Are you deliberately not defining a setter because you don't want others to be able to modify the private field? If so, writing to unreferencedTimestamp will silently do nothing. Would it be better to make others call getUnreferencedTimestamp() instead?

Reply:
Yeah, I made this field `readonly` in the interface definition and so didn't add a setter. I can change it to `getUnreferencedTimestamp()` but does that have additional value?

Reply:
I just wonder if it’s better for there to be no way for someone to modify the field than for there to _appear_to be a way to modify it. And the fact that writing to the underscore name modifies the field but writing to the non underscore name silently does nothing seems like it’s inviting a bug.

Reply:
Good point. I will update this.

Reply:
Added a single function to get all the data to be added to the summary.